RELEASE-NOTES.txt

```diff
@@ -5,6 +5,7 @@
 * [*] Comments can be filtered to show the most recent unreplied comments from other users. [#16215]
 
 * [*] Fixed the navigation bar color in dark mode. [#16348]
+* [**] A new author can be chosen for Posts and Pages on multi-author sites. [#16269, #16281, #16349]
```

Comment:
```suggestion
* [**] A new author can be chosen for Posts and Pages on multi-author sites. [#16281]
```
The PR link will mainly be used by beta testers testing the feature, so just referencing this PR should provide the needed testing steps for them.